cli/deployments/tekton.go

```diff
@@ -93,6 +80,20 @@ func (k Tekton) Delete(c *kubernetes.Cluster, ui *ui.UI) error {
 	}
 	if warning != "" {
 		ui.Exclamation().Msg(warning)
+		return nil
```

Comment:
Same as above, why exit on warnings? A warning can be anything, it should be more specific, otherwise this can result in unintended behavior.

Reply:
Added a comment and exiting only for a specific warning.